lib/std/Thread/RwLock.zig

```diff
@@ -41,7 +41,9 @@ pub fn tryLockShared(rwl: *RwLock) bool {
     return rwl.impl.tryLockShared();
 }
 
-/// Blocks until shared lock ownership is acquired.
+/// Obtains shared lock ownership is acquired.
+/// Blocks if another thread has exclusive ownership.
+/// Blocks if another thread is attempting to get exclusive ownership.
```

Comment:
The last line is more of an implementation choice ([read-preferring vs write-preferring](<https://en.wikipedia.org/wiki/Readers%E2%80%93writer_lock#Priority_policies>)). I don't think zig specifies which one it guarantees, especially since it defers to `pthread_rwlock_t` on some platforms.